styles.css

```diff
@@ -681,3 +681,93 @@ button:active {
   background-color: #2230d2;
   color: white;
 }
+
+h3 {
+  font-family: 'poppins', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+}
+
+section.contact__me__section {
+  background-image: url("image/contactformshapes-mobile@2x.png");
+  background-size: cover;
+  background-color: #6070ff;
+  background-repeat: no-repeat;
+  border-top-left-radius: 100px;
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.contact__me {
+  display: flex;
+  flex-direction: column;
+}
+
+.contact__me__title {
+  align-self: center;
+  width: auto;
+  margin-bottom: 5px;
+}
+
+.contact__me__text {
+  text-align: center;
+  font-family: 'poppins', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 25px;
+  color: #ebebff;
+  width: 285px;
+}
+
+.contact__me__form {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  border-radius: 50px;
+  width: auto;
+}
+
+.input {
+  width: 285px;
+  margin-top: 8px;
+  margin-bottom: 8px;
+  height: 48px;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
```

Comment:
- [ ] Let's make sure we add padding also in our inputs to match the Figma design 

Your implementation | Figma Design
:------:|:-----:
![image](https://user-images.githubusercontent.com/87689547/155786553-42c2d412-9018-4787-8c5e-84b0a3e88a96.png) | ![image](https://user-images.githubusercontent.com/87689547/155786626-28cb468b-2254-4603-b5bb-022c2b638b9f.png)